docs/cloud/faq/providers.md

```diff
@@ -0,0 +1,14 @@
+# Providers
+
+### Azure: Do you use Locally redundant storage (LRS) or Zone-redundant storage (ZRS) ?
+
+Customer data is only stored on Premium SSD block device targets. 
+We do not utilize Azure file or blob storage for customer data.
```

Comment:
**SIDENOTE:** I also saw it above. This new line will be rendered in the same line as the line above. If your intention is to have also a new line after rendering then you need to add an empty line.